solutions/64.js

```diff
@@ -0,0 +1,47 @@
+// Manik Sachdeva: msach22
+// reverse word order in a sentence
+// Input: "I am the king"
+// Output: "king the am I"
+
+/**
+* @param {string} sentence - string
+* @returns {string} newString - reversed string
+*/
+
+const reverseInput = (string) => {
+  let newString = '';
+  for (let i = string.length - 1; i >= 0; i--) {
+     newString += string[i];
+  };
+  return newString;
+}
+
+/**
+* @param {string} sentence - a sentence
+* @returns {string} result - reveresed word order
+*/
+
+const solution = (sentence) => {
+  if (sentence.length < 2) {
+    return "Incorrect Data Format";
+  }
+  let newString = reverseInput(sentence);
```

Comment:
why do you have so many `reverseInput`? 2 reverseInputs = same as starting point. I believe you could make this alot more efficient.